src/context/StoreContext.js

```diff
@@ -87,6 +87,20 @@ export default function StoreContextProvider(props) {
     localSetBibleReference(obj)
   }
 
+  const setNewBibleBook = (bId) => {
+    const newBibleBookSourceId = "new-Biblebook"
+    const _bibleRef = bibleReference
+    _bibleRef.sourceId = newBibleBookSourceId
```

Comment:
`newBibleBookSourceId` has only this as a callsite. Is this variable supposed to be config like? If not could we remove `newBibleBookSourceId`?

Reply:
I have added a preceding comment describing what it does

Reply:
I don't see your comment in the code